index.html

```diff
@@ -6,11 +6,9 @@
     <link rel="stylesheet" href="src/css/styles.css" type="text/css"/>
     <script src="http://fb.me/react-0.13.1.js"></script>
     <script src="http://fb.me/JSXTransformer-0.13.1.js"></script>
-    <script src="/socket.io/socket.io.js"></script>
-    <script src="https://cdn.appbase.io/scalr/appbase.js"></script>
+    <script src="bower_components/appbase-js/browser/appbase.js"></script>
```

Comment:
Wouldn't it be better if we use the url because if there is an update on the file it gets reflected here. Likewise it'll fail if there is a bug in the API. That way we'll know. what do you think ?